sysutils/progress/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        Xfennec progress 0.14 v
+github.tarball_from archive
+
+platforms           darwin
+categories          sysutils
+license             GPL-3
+
+maintainers         {gmail.com:herby.gillot @herbygillot} openmaintainer
+
+description         Tool to show progress for cp, mv, dd, ... (formerly known \
+                    as cv)
+
+long_description    This tool can be described as a Tiny, Dirty, \
+                    Linux-and-OSX-Only C command that looks for coreutils \
+                    basic commands (cp, mv, dd, tar, gzip/gunzip, cat, etc.) \
+                    currently running on your system and displays the \
+                    percentage of copied data. It can also show estimated \
+                    time and throughput, and provides a "top-like" mode \
+                    (monitoring).
+
+checksums           rmd160  8c2ff1cdf82f68601df483e303ccd8a1004b604b \
+                    sha256  214a0d84b3ee5dde57ec9952ec9aa68ad9261fb336ef025324b344ed7ab48af1 \
+                    size    58336
+
+build.target        ""
+destroot.args       PREFIX=${prefix}
+use_configure       no
```

Comment:
Hmm, the project doesn't specify any preference for any particular compiler.

Reply:
Did you read the wiki page I cited? You need to make sure the project is using the compiler that port determines is correct. Normally that is handled by the configure phase. Since there is no configure file, you need to manually make sure it gets the right compiler.

Reply:
My fault - I've addressed this.

Reply:
OK, change made